[core]/es_extended/server/classes/player.lua

```diff
@@ -460,7 +460,7 @@ function CreateExtendedPlayer(playerId, identifier, group, accounts, inventory,
 	end
 
 	function self.removeWeapon(weaponName)
-		local weaponLabel, weaponHash
+		local weaponLabel = nil
 		local weaponHash = joaat(weaponName)
         local playerPed = GetPlayerPed(self.source)
```

Comment:
Handle playerPed is nil, or weaponName is nil please add error message.

Reply:
added error message when playerPed is nil and take the hash from the loadout weapon name.